src/QsCompiler/BondSchemas/SandboxInput/SandboxInput.bond

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import "EntryPoint/EntryPoint.bond";
+namespace Microsoft.Quantum.QsCompiler.BondSchemas.SandboxInput;
+
+struct Bytecode
+{
+    5: blob Data;
+}
+
+struct Input
+{
+    5: Microsoft.Quantum.QsCompiler.BondSchemas.EntryPoint.EntryPointOperation EntryPoint;
+    10: bonded<Bytecode> QirBytes;
```

Comment:
Was there a reason wrap the `Blob` inside a struct rather than using `blob` directly here?

Reply:
Yeah, I guess you can only use `bonded` with structs. It might be fine to use a `blob` without `bonded` though? Not sure.

Reply:
I think you are right. In this case there is nothing to deserialize from just a `blob` field since it just contains "raw" bytes.  Then here it should be just a `blob` field.